src/main/java/com/google/api/generator/engine/format/BlockComment.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.format;
+
+import com.google.auto.value.AutoValue;
+import com.google.googlejavaformat.java.Formatter;
+
+@AutoValue
+public abstract class BlockComment implements Comment {
+  public abstract String comment();
+
+  public static Builder builder() {
+    return new AutoValue_BlockComment.Builder();
+  }
+
+  @AutoValue.Builder
+  public abstract static class Builder {
+    public abstract Builder setComment(String comment);
+
+    public abstract BlockComment build();
+  }
+
+  @Override
+  public String write() {
```

Comment:
PTAL at PR #29 and other examples / tests added since. As discussed, the `write()` method was used for prototyping and we've since moved to a Visitor pattern.

Reply:
moved to visitor pattern, thanks!